docs/upgrade.html

```diff
@@ -81,6 +81,8 @@ <h4><a id="upgrade_3_3_0" href="#upgrade_3_3_0">Upgrading a KRaft-based cluster
 
 <h5><a id="upgrade_330_notable" href="#upgrade_330_notable">Notable changes in 3.3.0</a></h5>
     <ul>
+        <li>KRaft mode is production ready for new clusters. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-833%3A+Mark+KRaft+as+Production+Ready">KIP-833</a>
```

Comment:
@ijuma should we use this URL https://cwiki.apache.org/confluence/x/8xKhD instead? This URL was generated using confluence share wiki feature.

Reply:
We don't use that for the other KIPs though. Are the main advantages that it's shorter and handles renames better?

Reply:
The URL I pasted will never change. The URL with the KIP name can change if the author decides to change the name of the KIP. I forget how Confluence performs the redirect. I think in some cases I have seen Confluence to it automatically in other cases I have seen it suggest a different URL in the rendered webpage.

Reply:
> We don't use that for the other KIPs though.

I use that kind of URL when referencing KIPs but maybe others don't.